android/app/src/main/cpp/fftools_ffmpeg.c

```diff
@@ -5450,10 +5450,11 @@ int ffmpeg_execute(int argc, char **argv)
         { "dn", OPT_BOOL | OPT_VIDEO | OPT_OFFSET | OPT_INPUT | OPT_OUTPUT, { .off = OFFSET(data_disable) },
             "disable data" },
 
-    #if CONFIG_VAAPI
-        { "vaapi_device", HAS_ARG | OPT_EXPERT, { .func_arg = opt_vaapi_device },
-            "set VAAPI hardware device (DRM path or X11 display name)", "device" },
-    #endif
+    // opt_vaapi_device is not defined
+    // #if CONFIG_VAAPI
+    //     { "vaapi_device", HAS_ARG | OPT_EXPERT, { .func_arg = opt_vaapi_device },
+    //         "set VAAPI hardware device (DRM path or X11 display name)", "device" },
+    // #endif
```

Comment:
What does this solves / fixes ?

Reply:
If I remove the comments the project does not compile because `opt_vaapi_device` is undefined. 
I could not find its definition in any header. So I guessed that on mobile `CONFIG_VAAPI` is not defined thus this part is never compiled.

Reply:
It looks like we might have to look at this again in the future to possibly reenable it correctly. Ok for now then.